client/js/components/hyperdrive/index.js

```diff
@@ -0,0 +1,39 @@
+const html = require('choo/html')
+var hyperdriveUI;
+if (!module.parent) hyperdriveUI = require('hyperdrive-ui')
+const path = require('path')
+const pretty = require('pretty-bytes')
+const getArchive = require('./archive.js')
+
+module.exports = function (state, prev, send) {
+  if (module.parent) {
```

Comment:
can we abstract the server-side markup below this line in the return block into a template file so it doesn't get browserified/packed up with static js (maybe put it in `/server/component/etc`) ? 

IMO this should never go to the client for any reason since it won't ever be used by client app.